package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vueuse/monorepo",
-  "version": "6.8.0",
+  "version": "6.7.6",
```

Comment:
👀

Reply:
Sry,sending a lot of conflicts, I will close this PR and resubmit 😂